src/vs/editor/common/editorContextKeys.ts

```diff
@@ -28,6 +28,7 @@ export namespace EditorContextKeys {
 
 	// -- mode context keys
 	export const languageId = new RawContextKey<string>('editorLangId', undefined);
+	export const editorExtension = new RawContextKey<string>('editorExtension', undefined);
```

Comment:
After a second look: we actually don't need to repeat this. Having `resourceExtname/Extension` is enough has it gets inherited to the editor from its column/slot. The reason for having `editorLangId` and `resourceLangId` is that latter is derived from the name while the former is more correct (look at the name and the contents)